### ISSUE TITLE
Bootstrap 5 support

### ISSUE THREAD

**Issue by fey** [laraeast/laravel-bootstrap-forms#3]
Hello! Bootstrap 5 is out. 

Will this package be updated?

**Comment by ahmed-aliraqi** [comment]
Of course, as soon as possible I will add Bootstrap 5 support, but unfortunately at the moment I am busy with other projects. 😥

**Comment by ahmed-aliraqi** [comment]
supported in v6.1.1 version
